gnuboardProject/backend/src/portfolio/portfolio.service.ts

```
@@ -96,81 +105,250 @@ export class PortfolioService {
+              if (!job) throw new NotFoundException('Job not found');
+              const portfolioJob = new PortfolioJob();
+              portfolioJob.section = savedSection;
+              portfolioJob.job = job;
+              await this.portfolioJobRepository.save(portfolioJob);
+            }
+            break;
+          case SectionType.Experience:
+            const experience = new PortfolioExperience();
+            experience.section = savedSection;
+            experience.company = String(sectionData.company ?? '');
+            experience.position = String(sectionData.position ?? '');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)